wiki/addons/first-party/kubejs-immersive-engineering/page.kubedoc

```diff
@@ -2,80 +2,99 @@ Download: [CurseForge](https://www.curseforge.com/minecraft/mc-mods/kubejs-immer
 
 ---
 
-Supported Recipe Types:
+# Supported Recipe Types
 
-- Alloy Kiln
+## Alloy Kiln
 ```js
 event.recipes.immersiveengineering.alloy(output, input1, input2)
 event.recipes.immersiveengineering.alloy(output, input1, input2, time)
 ```
-- Blast Furnace
+
+## Blast Furnace
 ```js
 event.recipes.immersiveengineering.blast_furnace(output, input)
 event.recipes.immersiveengineering.blast_furnace(output, input, slag)
 ```
-- Blast Furnace Fuel
+
+## Blast Furnace Fuel
 ```js
 event.recipes.immersiveengineering.blast_furnace_fuel(input, time)
 ```
-- Coke Oven
+
+## Coke Oven
 ```js
 event.recipes.immersiveengineering.coke_oven(output, input, creosote)
 event.recipes.immersiveengineering.coke_oven(output, input, creosote, time)
 ```
-- Garden Cloche
+
+## Garden Cloche
+>>> warn
```

Comment:
I think this would look better if it showed without the warn block, it looks confusing.
"## Garden Cloche -- This is currently broken on 1.19!"
That would look better ^